scipy/stats/tests/test_distributions.py

```diff
@@ -3258,6 +3258,14 @@ def test_ncx2_tails_pdf():
     assert_(np.isneginf(logval).all())
 
 
+def test_ncx2_zero_nc():
+    # gh-5441
+    # ncx2 with nc=0 is identical to chi2
+    result = stats.ncx2.cdf(1, nc=0, df=10)
+    expected = stats.chi2.cdf(1, df=10)
+    assert_equal(result, expected)
```

Comment:
please use something along the lines of `assert_allclose(result, expected, atol=1e-14)` for floating-point comparisons.

Reply:
Need also test pdf and logpdf. And I'd add at least a smoke test for an array_like `nc`, e.g. [4, 0].

Reply:
Please also add a test for `rvs`: recent numpies should be doing the right thing, however better check if our oldest supported numpy version is compatible.